src/components/discover-sheet/DiscoverSearch.js

```diff
@@ -58,6 +58,14 @@ export default function DiscoverSearch() {
     ensResults,
   ]);
 
+  const currencyListDataLength = useMemo(
+    () =>
+      uniswapCurrencyList?.[0]?.data?.length ||
+      0 + ensResults?.[0]?.data?.length ||
+      0,
```

Comment:
nit: can we do a string so theres no possibility of the length being the same? something like `${uniswapCurrencyList?.[0]?.data?.length}_${ensResults?.[0]?.data?.length}`

Reply:
yes, I don't want to be a pain in the ass, but this is not 100% unique

Reply:
💯  done!